spec/policies/hospital_policy_spec.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe HospitalPolicy do
+  let(:user) { create(:user) }
+  let(:hospital) { create(:hospital) }
+
+  describe "#index" do
+    context "when hospital has user" do
+      it { expect(described_class.new(user, hospital).index?).to be true }
+    end
+
+    context "when unregistered user" do
+      it { expect(described_class.new(nil, hospital).index?).to be false }
+    end
+  end
+
+  describe "#create" do
+    context "when hospital has user" do
+      it { expect(described_class.new(user, hospital).create?).to be true }
+    end
+
+    context "when unregistered user" do
+      it { expect(described_class.new(nil, hospital).create?).to be false }
+    end
+  end
+
+  describe "#update" do
+    context "when hospital has user" do
+      it { expect(described_class.new(user, hospital).update?).to be true }
```

Comment:
are user and hospital associated?

Reply:
My bad, sorry. Fixed! b0457a4adda1a2067db56f4d4a7937a82eb9be28